src/webln/NostrWeblnProvider.ts

```diff
@@ -364,6 +398,28 @@ export class NostrWebLNProvider implements WebLNProvider, Nip07Provider {
     );
   }
 
+  listTransactions(args: ListTransactionsArgs) {
+    this.checkConnected();
+
+    // maybe we can tailor the response to our needs
+    return this.executeNip47Request<
+      ListTransactionsResponse,
+      Nip47ListTransactionsResponse
+    >(
+      "list_transactions",
+      args,
+      (response) => !!response.transactions,
+      (response) => ({
+        transactions: response.transactions.map((transaction) => ({
+          ...transaction,
+          // NWC uses msats - convert to sats for webln
+          amount: Math.floor(transaction.amount / 1000),
+          fees_paid: Math.floor(transaction.fees_paid / 1000),
```

Comment:
we might need to be defensive there and make sure `transaction.amount` and `transaction.fees_paid` is available and a number.

Reply:
added defensive check on `fees_paid`